src/table/utils/selections-utils.ts

```diff
@@ -144,35 +161,91 @@ export const getSelectedRows = (
   return { ...selectedRows };
 };
 
+export const getMultiSelectedRows = (
+  selectedRows: Record<string, number>,
+  allRows: Row[],
+  evt: React.KeyboardEvent | React.MouseEvent,
+  cell: Cell,
+  firstCell?: Cell
+): Record<string, number> => {
+  const newSelectedRows = { ...selectedRows };
+
+  if ('key' in evt && evt.shiftKey && evt.key.includes('Arrow')) {
+    newSelectedRows[cell.qElemNumber] = cell.rowIdx;
+    // add the next or previous cell to selectedRows, based on which arrow is pressed
+    if (evt.key === 'ArrowDown') {
```

Comment:
Another suggestion might be of having an array with arrow key codes;

`const arrowKeyCodes = [KeyCodes.ARROW_DOWN,KeyCodes.ARROW_UP...etc]`

then in line 173 `arrowKeyCodes.includes(evt.key)` instead of `evt.key.includes('Arrow')`

```suggestion
    if (evt.key === KeyCodes.ARROW_DOWN) {
```

Reply:
ahh, yes so one enum with all used keycodes and then another array with arrow keys. I like it. Will do this in another PR right away